src/datalink/mod.rs

```diff
@@ -57,6 +56,8 @@ pub mod pcap;
 
 pub mod dummy;
 
+/// Type alias for an EtherType.
+pub type EtherType = u16;
```

Comment:
Doesn't look like this is used anywhere?

Reply:
Because it has the same name, so it's not part of the diff:
```rust
pub enum ChannelType {
    /// Send and receive layer 2 packets directly, including headers
    Layer2,
    /// Send and receive "cooked" packets - send and receive network layer packets
    Layer3(EtherType),
}
```